app/controllers/foreman_gutterball/api/v2/content_reports_controller.rb

```diff
@@ -4,29 +4,46 @@ module V2
       class ContentReportsController < ::Katello::Api::V2::ApiController
         api :GET, '/content_reports', 'List available reports'
         def index
-          service.available_reports
+          @report = service.report_details
```

Comment:
it's not particularly necessary for you to use class variables for the reports because there aren't any other methods that will need them in between when you fill the variable and when you return it for the response. #nitpick